ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -90,10 +91,10 @@
   private long chainId;
 
   @Option(
-      names = {"--data-directory"},
-      description = "Data directory to store temporary files",
+      names = {"--data-path"},
+      description = "The path to EthSigner data directory to store temporary files",
```

Comment:
reword (nit): Path to a directory for storage of temporary files?rewo

Reply:
done